files/english/ORGANIZATION-DAY.md

```diff
@@ -1 +1,22 @@
-# Organization on the day
\ No newline at end of file
+# Organization on the day
+
+>Here are some instructions and tips to be followed by the PUG-PE organizing members and volunteers on the event's day.
+
+## Instructions
+
+### Some things to be aware be of before the event's day
+
+It is important to the organizing member and volunteers to check if there are some items that might be necessary, just like: table, power plugs, video projector. It is wanted that, in case those items are not available, the members should look for alternate solutions so the meetup can happen in the best possible way for everyone.
+
+---
+
+### Show up early
+
+The one who are responsible for the organization of the meet up need to set up the event site, so, show up early! This way you can solve any unexpected issue calmly, set up the chairs, the video projector, greet the speakers and the participants.
+Lots of times the meetups happen, for example, in universities classrooms, and, most likely, not all participants know the site and end up getting lost and not finding the room where the event will be held. So, indicate well where the event will happen!
+
+---
+
+### PUG-PE introduction
+
+The PUG-PE is an event that is always welcoming new participants and lots of them, besides participating on the meet up, do not know the Python local community and, sometimes, not even what a community is. That's why it is really important that there is a brief introduction, lasting between 5 and 10 minutes, introducing what is and what are the goals of a community and the Python local community.
```

Comment:
Não entendi a última sentença: "what is and what are the goals of a community and the Python local community".

Tu poderia explicar?

"what is" o que? "Python local community"? O que acho que entendi foi que a conexão entre as partes da sentença ficou quebrada e por isso não tá claro.